certbot/CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
   every certificate in the chain.
   See [#8577](https://github.com/certbot/certbot/issues/8577).
 * Support for Python 2 has been removed.
+* certbot-auto will stop receiving updates on systems other than those based on
+  Debian or RHEL (excluding Amazon Linux who stopped receiving updates in this
```

Comment:
the multiple nots make this confusing and I'd like to be exceedingly clear what's going on here.

How about:

"This release, we are beginning to roll out changes to the certbot-auto script to stop receiving any updates. This release includes Amazon Linux users, and all other systems that are not based on Debian or RHEL. We plan to make this change to the certbot-auto script for all users in the coming months."

Reply:
Good idea. I added another sentence and tweaked the beginning of your first sentence slightly, but I otherwise made this change.